codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/auth/SigV4AsymmetricAuthSchemeIntegration.kt

```diff
@@ -25,7 +25,8 @@ import software.amazon.smithy.model.shapes.ShapeId
  * Register support for the `aws.auth#sigv4a` auth scheme.
  */
 class SigV4AsymmetricAuthSchemeIntegration : KotlinIntegration {
-    override val order: Byte = -59
+    // Needs to happen after the `SigV4AsymmetricTraitCustomization` (-60).
+    override val order: Byte = -50
```

Comment:
**Question**: This now has the same order (`-50`) as `SigV4AuthSchemeIntegration`...is that intentional?

Reply:
Yeah, the order doesn't really matter between those two anymore, the `order` being the same number isn't really an issue either. They're sorted by `sortedBy(KotlinIntegration::order)`